db_revisions/versions/6613e1e2c133_replace_is_active_with_lei_status_in_fi.py

```diff
@@ -0,0 +1,92 @@
+"""Replace is_active with lei_status in financial_institutions table
+
+Revision ID: 6613e1e2c133
+Revises: 6dd77f09fae6
+Create Date: 2024-11-13 00:43:53.489086
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+from db_revisions.utils import table_exists, get_table_by_name, get_indices_from_collection
+
+
+# revision identifiers, used by Alembic.
+revision: str = "6613e1e2c133"
+down_revision: Union[str, None] = "6dd77f09fae6"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+lei_status_seed_data = [
+    {"code": "ISSUED", "name": "Issued", "can_file": True},
+    {"code": "PUBLISHED", "name": "Published", "can_file": False},
+    {"code": "DUPLICATE", "name": "Duplicate", "can_file": False},
+    {"code": "LAPSED", "name": "Lapsed", "can_file": False},
+]
```

Comment:
only ones we care about are `ISSUED`, `RETIRED`, and `LAPSED`; `PUBLISHED`, and `DUPLICATE` are not in the set mentioned.